packages/components/src/form-token-field/style.scss

```
@@ -142,7 +141,7 @@
 
 .components-form-token-field__remove-token.components-button {
 	cursor: pointer;
-	border-radius: 0 2px 2px 0;
+	border-radius: 0 $radius-x-small $radius-x-small 0;
```

Instead of re-setting all border radiii, should we instead just set top left and bottom left to 0? Using logical properties:

```suggestion
  // Remove top-left and bottom-left border radii
	border-start-start-radius: 0;
	border-end-start-radius: 0;
```

---

Ignore my previous comment. I misunderstood how the radius is applied